skbase/base/_base.py

```diff
@@ -449,20 +430,41 @@ def clone_tags(self, estimator, tag_names=None):
         Changes object state by setting tag values in tag_set from estimator as
         dynamic tags in self.
         """
-        tags_est = deepcopy(estimator.get_tags())
+        self._clone_flags(
+            estimator=estimator, flag_names=tag_names, flag_attr_name="_tags"
+        )
 
-        # if tag_set is not passed, default is all tags in estimator
-        if tag_names is None:
-            tag_names = tags_est.keys()
-        else:
-            # if tag_set is passed, intersect keys with tags in estimator
-            if not isinstance(tag_names, list):
-                tag_names = [tag_names]
-            tag_names = [key for key in tag_names if key in tags_est.keys()]
+        return self
 
-        update_dict = {key: tags_est[key] for key in tag_names}
+    def get_config(self):
```

Comment:
Not a blocking change, but just noting that later we'll want to generalize `get_config` to get global config and local config (currently only handles local config).

Reply:
hm, I thought the future logic was going to be, local config overrides global config, and the return is `global.update(local)`

Reply:
@fkiraly that's actually what I meant. We'll need an extension point so that we users can specify global config locations that implement something that aligns with the protocol (e.g, can point to their own packages global get_config). The idea would  be to get skbase global configs, user global configs and then local configs. 

Analagous to walking the MRO  and collecting tags, but walking specified global configs.